core/fedavgoptimizer.py

```diff
@@ -0,0 +1,144 @@
+from core.utils.event_types import ActionableEventTypes, RawEventTypes, callback_handler
+from core.utils.dmljob import deserialize_job
+from core.utils.dmlresult import DMLResult
+
```

Comment:
Can you add an extra line here? Our standard is two empty lines between import statements and rest of code.

Reply:
@neeleshdodda44 wasn't sure since this standard isn't followed in rest of codebase